pyanaconda/modules/payloads/source/harddrive/harddrive.py

```diff
@@ -64,6 +64,15 @@ def description(self):
         """Get description of this source."""
         return "{}:{}".format(self._device, self._directory)
 
+    def __repr__(self):
+        result = "Source(type='HDD', partition='{}', directory='{}', install_tree_path='{}')" \
```

Comment:
I am not sure if there should `install_tree_path` in the string. What do you think, @jkonecny12?

Reply:
I agree it should be removed, all other sources have only "inputs" displayed there.

Reply:
Removed.